bensconv.py

```diff
@@ -34,3 +34,123 @@ def conv_uniform_R(obspec,modspec,R):
     
 
     return fluxout
+    
+    
+    
+    
+##############################################################################
+##############################################################################
+##############################################################################
+##############################################################################
+##############################################################################
+##############################################################################
+
+
+
+
+#### CONVOLVING THE MODEL SPECTRA WITH THE NON-UNIFORM RESOLVING POWER (NIRSPEC + MIRI)
+
+#wl =  obs_data[:, 0]  #np.arange(1,16,1)   lambda in microns
+
+
+
+def JWST_R(wl):
+    
+    '''Function that provides the Resolving Power (R) as a function of wavelength (wl) in micron,
+       depending on jwst instrument'''
```

Comment:
Add a more descriptive docstring that describes `wl` and the different possible things that this function returns.  Follow the  [numpy example](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) function `def function_with_types_in_docstring`